docs/develop/python/message-passing.mdx

```diff
@@ -317,6 +317,60 @@ To obtain an Update handle, you can:
 - Use [`start_update`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#start_update) to start an Update and return the handle, as shown in the preceding example.
 - Use [`get_update_handle_for`](https://python.temporal.io/temporalio.client.WorkflowHandle.html#get_update_handle_for) to fetch a handle for an in-progress Update using the Update ID.
 
+
+#### Update-With-Start {#update-with-start}
+
+:::tip Stability
+
+In [Public Preview](/evaluate/development-production-features/release-stages#public-preview) in Temporal Cloud.
+
+Minimum Temporal Server version [Temporal Server version 1.26](https://github.com/temporalio/temporal/releases/tag/v1.26.0)
```

Comment:
I had fixed this yesterday; the tag is wrong: https://github.com/temporalio/documentation/commit/d54486d125e828e4d0073c67c09f370f73c8dad7

Reply:
Oh, it seems I'm still using the wrong one in Java and Go

Reply:
Yep, done